components/products/ProductFilter.tsx

```diff
@@ -0,0 +1,67 @@
+import { Dispatch, FC, forwardRef, SetStateAction, useEffect } from "react";
+import { useForm, UseFormRegisterReturn } from "react-hook-form";
+
+const CheckboxFilter = forwardRef<HTMLInputElement, { item: string } & UseFormRegisterReturn<string>>(
+  ({ item, ...props }, ref) => {
+    return (
+      <label className="checkbox-container">
+        <input {...props} ref={ref} type="checkbox" />
+        <div className="checkmark" />
+        <div className="text" data-text={item} />
+      </label>
+    );
+  }
+);
+
+const checkboxes = ["Espresso", "Beans", "Pods", "Frothers", "Grinders", "Accessories", "Cups"] as const;
```

Comment:
@BabisDigos  ❓why are those capitalised ?

Reply:
In DB I have them like that, and I was bored changing them.